packages/client-search/src/types/BatchActionType.ts

```diff
@@ -4,11 +4,15 @@ export const BatchActionEnum: Readonly<Record<string, BatchActionType>> = {
   PartialUpdateObject: 'partialUpdateObject',
   PartialUpdateObjectNoCreate: 'partialUpdateObjectNoCreate',
   DeleteObject: 'deleteObject',
+  DeleteIndex: 'delete',
```

Comment:
@Ant-hem @aseure can you quick check those in your clients?

Reply:
What do we need to check? If batch actions do accept `delete`? Could you open issues with `polyglot` with the exact context?

Reply:
@aseure Correct. I will do it.

Reply:
@nunomaduro I've just checked and it's available in Go, Scala, Java, C#.

Reply:
can we merge this one as everybody seems to agree ? ☺️